src/components/logs/Line.tsx

```diff
@@ -35,19 +35,14 @@ function LogLine({ line, lineNumber, disableSelect }: Props) {
                 },
             }}
         >
-            <Stack
-                direction="row"
-                spacing={2}
-                sx={{
-                    alignItems: 'center',
-                }}
-            >
+            <Stack direction="row" spacing={2}>
                 <Box
                     sx={{
                         color: lineNumberColor,
                         userSelect: 'none',
                         minWidth: 50,
                         textAlign: 'right',
+                        pt: 0.5,
```

Comment:
Making the log lines indent so now we need to keep the row numbers at the top of the element but pushed down a tiny bit.